app/views/hub/documents/confirm.html.erb

```diff
@@ -0,0 +1,39 @@
+<% @title = t(".title") %>
+<% content_for :page_title, @title %>
+<% content_for :card do %>
+  <div class="slab">
+    <div class="grid-flex full-height">
+      <div class="item">
+        <h1><%= t("hub.documents.confirm.title", document_type: @document.document_type) %></h1>
+        <p>
+          <%= t("hub.documents.confirm.text",
+                document_type: @document.document_type,
+                client_name: @document.client.legal_name,
+                filing_year: @document.tax_return.year
+                )
+          %>
+        </p>
+        <div class="grid-flex">
+          <div class="item-15r">
+            <%= link_to hub_client_documents_path(client_id: @document.client.id), class: "button button--icon button--icon--centered", "data-track-click": "document-confirmed-yes" do %>
+              <%= image_tag("checkmark.svg", alt: "") %><%= t("general.affirmative") %>
+            <% end %>
+          </div>
+          <div class="item">
+            <%= link_to edit_hub_client_document_path(id: @document.id), class: "button button--icon button--icon--centered", "data-track-click": "document-confirmed-no" do %>
```

Comment:
The doc will already be saved at this point. Will Hub users know what to do to archive the doc? Is it the case that clients will never see archived docs?